Releases notes NethServer 7.8.2003 (#498)
NethServer/docs: administrator-manual/en/release_notes.rst

.. only:: nsent

    - ISO release 7.8.2003 "final" replaces any previous ISO 7.7.1908

    - This release is based on `CentOS 7 (1908) <https://wiki.centos.org/Manuals/ReleaseNotes/CentOS7>`_

    - CentOS 7 will receive security updates until 2024-06-30

Major changes on 2020-xx-05
---------------------------

* ISO release 7.8.2003 "final" replaces any previous ISO 7.7.1908

* The new Server Manager implementation based on Cockpit is now marked as stable

> * On new installations, ``Override the shell of users`` option is enabled by default.

Reviewer: ```suggestion
* On new installations, the :guilabel:`System > Settings > Shell policy > Override the shell of users` option is enabled by default.
```

>   Normal users will be able to login into the new Server Manager only if ``Enable user settings page`` option has been enabled, or if the user have been

Reviewer: ```suggestion
  Normal users will be able to log in to the new Server Manager only if :guilabel:`System > Settings > User settings page > Enable user settings page` option has been enabled, or if the user has been
```

  delegated to access at least one module.

>   SSH access is limited by default to ``root`` and users inside the administrator group

Reviewer: ```suggestion
  SSH access is limited to ``root`` and users inside the designed administrative group (``Domain Admins`` by default)
```

Author: I do not like proposed change, I think that "by default" should be stated because the behavior can be changed.

Author: ```suggestion
  SSH access is limited by default to ``root`` and users inside the designed administrative group (``Domain Admins`` by default)
```

Reviewer: I don't like the repetition but it's OK anyway

Author: What about:
```
SSH access is limited to ``root`` and users inside the designed administrative group (``Domain Admins`` by default). More granular permissions can be tuned from :guilabel:`SSH` page.
```
?

> * All IMAP actions will be logged by default into ``/var/log/maillog``

Reviewer: ```suggestion
* All IMAP actions will be logged by default into ``/var/log/imap``
```
